package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Simple HTML5 drag-drop zone with React.js",
   "main": "lib/index.js",
   "scripts": {
-    "build": "babel ./src --out-dir ./lib",
+    "build": "babel ./src --out-dir ./lib && webpack && uglifyjs ./dist/react-dropzone.js --compress --mangle --output ./dist/react-dropzone.min.js",
```

Comment:
The point of using webpack is to have the whole compiling pipeline set up with it. Otherwise it doesn't make any sense. Can you switch to webpack with uglify plugin instead of using uglifyJS directly?